apex_legends/exceptions.py

```diff
@@ -19,4 +19,4 @@ def __init__(self, message='The specified player was not found'):
 class ServerError(BaseException):
 
     def __init__(self, message='There is an error with the server.'):
-        super().__init__()
+        super().__init__()
```

Comment:
Not required

Reply:
I'm unsure as to why these changes occured, but as I said earlier, blame REPL.it for this.